utilities/chartColors.js

```diff
@@ -0,0 +1,20 @@
+const colors = {
```

Comment:
Nice, hardcoding it is definitely a great option with a small dataset like we have.